docs/sources/metainfo/component.xml

```diff
@@ -567,6 +567,10 @@
 				<literal>recommends</literal> tag should be used. In this case, AppStream clients should allow the installation of the software
 				component, but may display a warning before allowing it.
 			</para>
+			<para>
+				To specify that a device is being used, an application can use the <literal>supports</literal> tag. This will mean that a certain requirement
+				will be integrated successfully. This is useful in case the client wants to list components that support a certain feature exclusively.
```

Comment:
> This will mean that a certain requirement will be integrated successfully.

That sounds super weird. Also, whether something happens successfully or not is not something the spec knows. Maybe "This means that the software component supports a certain configuration or other item.".
But I am not sure yet whether `supports` is such a great idea yet.

Btw, please write in present tense for specification changes (it described the status quo, and rarely what will be done in future).